perun-core/src/main/java/cz/metacentrum/perun/core/impl/Auditer.java

```diff
@@ -144,54 +143,46 @@ public AuditMessage mapRow(ResultSet rs, int i) throws SQLException {
 		}
 	};
 
-	protected static final RowMapper<AuditMessage> AUDITMESSAGE_MAPPER_FOR_PARSER = new RowMapper<AuditMessage>() {
-		public AuditMessage mapRow(ResultSet rs, int i) throws SQLException {
+	protected static final RowMapper<AuditMessage> AUDITMESSAGE_MAPPER_FOR_PARSER = (rs, i) -> {
```

Comment:
I have a little problem with this kind of replacements. Its not easily recognizable, what method is actually implemented. In Intellij IDEA you can get it displayed only by hovering/clicking on brackets. It would be better, if we had at least full name of parameters passed, like ``(resultSet, rowNumber)`` or whatever instead of ``(rs, i)``.